crates/hyperswitch_connectors/src/connectors/worldpay/requests.rs

```diff
@@ -284,3 +330,6 @@ pub struct WorldpayCompleteAuthorizationRequest {
     #[serde(skip_serializing_if = "Option::is_none")]
     pub collection_reference: Option<String>,
 }
+
+pub const THREE_DS_MODE: &str = "always";
```

Comment:
Nit: These need not be `pub`, can be `pub(super)`, they should still be accessible from within the `worldpay` module.